src/data/pages/tos.de.md

```diff
@@ -0,0 +1,229 @@
+
+### Datenweitergabe
+
+Im Rahmen der Erbringung unserer Dienstleistungen, sowie der Bereitstellung unserer Webapplikation, sind wir unter Umständen darauf angewiesen, die Dienste Dritter in Anspruch zu nehmen. In diesem Zusammenhang kann es vorkommen, dass wir Dritte mit der Bearbeitung deiner personenbezogenen Daten beauftragen. In diesem Fall stellen wir sicher, dass diese Dritten die Voraussetzungen des Datenschutzes einhalten. Unter Umständen sind wir auch verpflichtet, deine Daten Behörden oder weiteren Dritten bekanntzugeben.
+
+Folgende Dritte können oder sind im Rahmen der Erbringungen involviert (siehe Kapitel Begriffe für nähere Erklärungen):
+
+* Hosting-Plattform-Betreiberin DigitalOcean
+* Spamschutz und DNS-Server Cloudflare
+* E-Mail-Services Sendgrid und ImprovMX / Reflectiv SAS
+* Google, via reCAPTCHA, wenn du das _eCamp Core Team_ via Mail oder Kontaktformular kontaktierst, und bei allfälligem Login via Google
+* hitobito / Puzzle ITC bei allfälligem Login via MiData oder cevi.db
+* Sentry, bei allfällig auftretenden Fehlern in der Software
+
+Wir geben deine personenbezogenen Daten nur weiter, wenn eine der folgenden Voraussetzungen erfüllt ist:
+
+* Deine Einwilligung liegt vor;
+* es besteht eine gesetzliche Verpflichtung;
+* wenn dies für die Funktionalität der von dir genutzten Features erforderlich ist;
+* wenn dies zur Durchsetzung unserer Rechte, insbesondere Durchsetzung von Ansprüchen aus dem Nutzungsbedingungen, erforderlich ist;
+* wenn wir daran ein berechtigtes Interesse haben und deine gegenteiligen Interessen nicht überwiegen;
+* wenn eine andere gesetzliche Erlaubnis vorliegt.
+
+Einige der genutzten Dienste gehören zu Unternehmen im Ausland. Daher kann es im Rahmen der Auftragsverarbeitung auch zu Übermittlung deiner personenbezogenen Daten an diese Unternehmen im Ausland kommen. Diese Unternehmen sind im gleichen Umfang zum Datenschutz verpflichtet, wie wir selber. Entspricht das Datenschutzniveau in dem Land, in welches Daten übertragen werden, nicht demjenigen der Schweiz oder der Europäischen Union, so stellen wir sicher, dass der gleiche Schutz wie in der Schweiz bzw. der Europäischen Union garantiert wird.
+
+Kommt es zu einer Datenübermittlung an ein Unternehmen in den USA, stellen wir sicher, dass dieses Unternehmen gemäss dem Swiss- bzw. EU-US-Privacy Shield Abkommen zertifiziert ist und damit sichergestellt wird, dass das Datenschutzniveau der Schweiz bzw. EU eingehalten wird. Fehlt eine Zertifizierung, holen wir die notwendigen Garantien vertraglich ein.
```

Comment:
Können wir das sicherstellen?

Reply:
Wir müssen, soweit ich weiss. Ist nach meinem (limitierten) Verständnis ein wichtiger Teil des EU-Datenschutzgesetzes und des neuen CH-Datenschutzgesetzes.

Reply:
* [digitalocean](https://www.privacyshield.gov/participant?id=a2zt0000000TQNgAAO&status=Active), [Cloudflare](https://www.privacyshield.gov/participant?id=a2zt0000000GnZKAA0&status=Active), [Google](https://www.privacyshield.gov/participant?id=a2zt000000001L5AAI&status=Active) und [Sentry](https://www.privacyshield.gov/participant?id=a2zt0000000YdenAAC&status=Active) sind auffindbar auf der [Liste](https://www.privacyshield.gov/list) von US-Unternehmen welche sich zur Einhaltung des Swiss-U.S. Privacy Shield vertraglich verpflichtet haben
* SendGrid gehört anscheinend zu [Twilio](https://www.privacyshield.gov/participant?id=a2zt0000000TNLbAAO&status=Active), welche sich ebenfalls zur Einhaltung verpflichtet haben, und SendGrid wird auf ihrem Eintrag namentlich nochmals erwähnt.
* ImprovMX gehört der französischen Firma Reflectiv SAS, welche somit unter das EU-Datenschutzgesetz fällt.

I'm not a lawyer, aber da würde ich jetzt mal abwarten was die juristische Kommission der PBS dazu meint.